src/app/routes/learnerRoutes.js

```diff
@@ -229,21 +233,21 @@ function proxyObj (){
         return require('url').parse(learnerURL + urlParam).path
       }
     },
-    userResDecorator: function (proxyRes, proxyResData,  req, res) {
+    userResDecorator: function (proxyRes, proxyResData, req, res) {
       try {
-        logger.info({msg: 'proxyObj'});
+        logger.info({ msg: 'proxyObj' });
```

Comment:
Make sure you use logger.debug for dev testing.. 

You should remove the logs later after testing.